classification/recipe.yaml

```diff
@@ -30,9 +30,7 @@ steps:
     # Specifies the method that defines the data transformations to apply during model inference
     transformer_method: transformer_fn
   train:
-    using: custom
-    # Specifies the method that defines the estimator type and parameters to use for model training
-    estimator_method: estimator_fn
```

Comment:
Do we want to remove or empty the file where `estimator_fn` is mentioned + remove the tests if any?